include/d/actor/d_a_leaflift.h

```diff
@@ -2,29 +2,66 @@
 #define D_A_LEAFLIFT_H
 
 #include "f_op/f_op_actor.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_cc_d.h"
+#include "d/d_bg_w.h"
+#include "d/d_particle.h"
+
+class dBgW;
 
 class daLlift_c : public fopAc_ac_c {
 public:
     bool _delete();
-    void CreateHeap();
+    BOOL CreateHeap();
     void CreateInit();
     s32 _create();
     void set_mtx();
     void setMoveBGMtx();
     bool _execute();
     void emitterCtrl();
-    void MoveDownLift();
+    BOOL MoveDownLift();
     bool _draw();
 
-    void checkEndDownLift();
-    void MoveUpLift();
+    s32 checkEndDownLift();
+    s32 MoveUpLift();
 
     static const f32 m_height;
     static const f32 m_max_speed;
     static const f32 m_min_speed;
 
+    static const char m_arcname[6];
+
 public:
-    /* Place member variables here */
+    /* 0x290 */ request_of_phase_process_class mPhs;
+    /* 0x298 */ J3DModel* mpModel;
+    /* 0x29C */	dCcD_Stts mStts;
+    /* 0x2D8 */ dCcD_Cyl mCyl;
+    /* 0x408 */ dBgW* mcBgW;
```

Comment:
This is a dBgW not a cBgW.

I normally name these `mpBgW` since the c/d prefix just indicates which library they're a part of and don't add any information in this context (c = SSystem library, d = dolzel aka zelda-specific subclasses)